tests/validation/tests/v3_api/test_import_k3s_cluster.py

```diff
@@ -1,19 +1,39 @@
 import os
-from lib.aws import AmazonWebServices
 from .common import *  # NOQA
+from lib.aws import AmazonWebServices
+from python_terraform import *
+
 
 DATA_SUBDIR = os.path.join(os.path.dirname(os.path.realpath(__file__)),
                            'resource')
-RANCHER_K3S_VERSION = os.environ.get("RANCHER_K3S_VERSION", "")
+RANCHER_REGION = os.environ.get("AWS_REGION")
+RANCHER_VPC_ID = os.environ.get("AWS_VPC")
+RANCHER_SUBNETS = os.environ.get("AWS_SUBNET")
+RANCHER_AWS_AMI = os.environ.get("AWS_AMI", "")
+RANCHER_AWS_USER = os.environ.get("AWS_USER", "ubuntu")
+AWS_SSH_KEY_NAME = os.environ.get("AWS_SSH_KEY_NAME", "")
 HOST_NAME = os.environ.get('RANCHER_HOST_NAME', "sa")
+
+RANCHER_RESOURCE_NAME = os.environ.get("RANCHER_RESOURCE_NAME", "")
+RANCHER_K3S_VERSION = os.environ.get("RANCHER_K3S_VERSION", "")
+RANCHER_K3S_NO_OF_SERVER_NODES = \
+    os.environ.get("RANCHER_K3S_NO_OF_SERVER_NODES", 3)
 RANCHER_K3S_NO_OF_WORKER_NODES = \
-    os.environ.get("RANCHER_AWS_NO_OF_WORKER_NODES", 3)
+    os.environ.get("RANCHER_K3S_NO_OF_WORKER_NODES", 3)
+RANCHER_K3S_SERVER_FLAGS = os.environ.get("RANCHER_K3S_SERVER_FLAGS", "server")
+RANCHER_K3S_WORKER_FLAGS = os.environ.get("RANCHER_K3S_WORKER_FLAGS", "agent")
+RANCHER_QA_SPACE = os.environ.get("RANCHER_QA_SPACE", "")
+
+RANCHER_EXTERNAL_DB = os.environ.get("RANCHER_EXTERNAL_DB", "mysql")
+RANCHER_EXTERNAL_DB_VERSION = os.environ.get("RANCHER_EXTERNAL_DB_VERSION", "5.7")
+RANCHER_INSTANCE_CLASS = os.environ.get("RANCHER_INSTANCE_CLASS", "db.t2.micro")
+RANCHER_DB_GROUP_NAME = os.environ.get("RANCHER_DB_GROUP_NAME", "default.mysql5.7")
+RANCHER_DB_USERNAME = os.environ.get("RANCHER_DB_USERNAME", "adminuser")
+RANCHER_DB_PASSWORD = os.environ.get("RANCHER_DB_PASSWORD", "adminpassword")
```

Comment:
Having a default password is not secure, especially as this is in a public repo.